flink-tests/src/test/scala/org/apache/flink/api/scala/util/CollectionDataSets.scala

```diff
@@ -55,7 +55,6 @@ object CollectionDataSets {
     data.+=((19, 6L, "Comment#13"))
     data.+=((20, 6L, "Comment#14"))
     data.+=((21, 6L, "Comment#15"))
-    Random.shuffle(data)
```

Comment:
I think this should be added back.

Reply:
No, the result of this shuffle is discarded. `data` is not modified.